app/controllers/predicates/AuthPredicate.scala

```diff
@@ -97,7 +97,7 @@ class AuthPredicate @Inject()(authService: EnrolmentsAuthService,
           case _ => insolvencySubscriptionCall(user, block)
         }
       case None =>
-        logger.debug("[AuthPredicate][authoriseAsNonAgent] - Non-agent with no HMRC-MTD-VAT enrolment. Rendering unauthorised view.")
```

Comment:
same here

Reply:
i think this should be kept as errorLog since the user is kicked out of the journey for not having an enrolment